template/package.json

```
@@ -11,23 +11,23 @@
     "generate": "nuxt generate"
   },
   "dependencies": {
-    "nuxt": "^2.1.0",
-    "vuetify": "^1.2.6"
+    "nuxt": "^2.2.0",
+    "vuetify": "^1.3.2"
   },
   "devDependencies": {
     "babel-eslint": "^7.2.3",
     {{#alacarte}}
-    "babel-plugin-transform-imports": "^1.4.1",
+    "babel-plugin-transform-imports": "^1.5.1",
     {{/alacarte}}
-    "eslint": "^4.3.0",
-    "eslint-config-standard": "^10.2.1",
-    "eslint-loader": "^1.9.0",
-    "eslint-plugin-html": "^3.1.1",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-node": "^5.1.1",
-    "eslint-plugin-promise": "^3.5.0",
-    "eslint-plugin-standard": "^3.0.1",
+    "eslint": "^5.7.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-loader": "^2.1.1",
+    "eslint-plugin-html": "^4.0.6",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-node": "^7.0.1",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-standard": "^4.0.0",
```

```suggestion
    "eslint-plugin-standard": "^4.0.0",
    "eslint-plugin-vue": "^4.0.0",
```

Maybe consider adding a Vue linter? 
https://github.com/vuejs/eslint-plugin-vue

---

we are using standard eslint javacript rules eslint-plugin-vue required when we use prettier and use plugin:vue/essential or recommend i suggest to follow the standard javscript eslint rules